bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -56,7 +87,13 @@ def get_outputs(self, output_ids: List[OutputId]) -> List[OutputWithMetadata]:
 
     def get_outputs_ignore_errors(self, output_ids: List[OutputId]) -> List[OutputWithMetadata]:
         """Try to get OutputWithMetadata from provided OutputIds.
-           Requests are sent in parallel and errors are ignored, can be useful for spent outputs.
+        Requests are sent in parallel and errors are ignored, can be useful for spent outputs.
+
+        Args:
+            output_ids (List[OutputId]): list of output ids
```

Comment:
Technically (just read from spec) if we use typing, we should omit types in descriptions

Reply:
Should be done unless I forget something.